static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx

```diff
@@ -56,7 +58,8 @@ export function OnboardingLayout({
   const {platformOptions} = docsConfig;
 
   const {introduction, steps, nextSteps} = useMemo(() => {
-    const {onboarding} = docsConfig;
+    const {onboarding, replayOnboarding} = docsConfig;
+    const doc = isReplayOnboarding && replayOnboarding ? replayOnboarding : onboarding;
```

Comment:
couldn't figure out a nicer way to do this, but this is what outputs the replay onboarding docs instead of the normal docs